weaviate/data/crud_data.py

```diff
@@ -430,14 +436,20 @@ def get(
         additional_properties : list of str, optional
             list of additional properties that should be included in the request,
             by default None
-        with_vector: bool
+        with_vector : bool
             If True the `vector` property will be returned too,
             by default False
         class_name : Optional[str], optional
             The class name of the object with UUID `uuid`. Introduced in Weaviate version v1.14.0.
             STRONGLY recommended to set it with Weaviate >= 1.14.0. It will be required in future
             versions of Weaviate Server and Clients. Use None value ONLY for Weaviate < v1.14.0,
             by default None
+        consistency_level : Optional[str], optional
+            Can be one of 'ALL', 'ONE', or 'QUORUM'. Determines how many replicas must acknowledge
```

Comment:
You could use string Enums instead
```
class ConsistencyLevel(str, Enum):  # can be replaced with StrEnum in Python 3.11
    ALL = auto()
    ONE = auto()
    QUORUM = auto()
```

the type would then be `Optional[ConsistencyLevel]`. Users can then use 
`get(...,ConsistencyLevel.QUORUM)`or just `"QUORUM"` (as a string, just like now). This gives typesafety for advanced users and users that don't care can still use strings

Reply:
Sure, done 👍